styles/style.css

```diff
@@ -0,0 +1,1028 @@
+.menu li:nth-of-type(1) {
+    flex-basis: 60px;
+    /* border: red 2px solid;  */
+    position: relative;
+    transform: translateX(-57px)
+  }
+
+  .menu li:nth-of-type(2) {
+     flex-basis: 5px;
+   /*  border: red 2px solid;  */
+    position: relative;
+    transform: translateX(-50px)
+  }
+  .menu li:nth-of-type(3) {
+   flex-basis: 60px;
+   /*  border: red 2px solid;  */
+    position: relative;
+    transform: translateX(-53px)
+  }
+
+  .menu li:nth-of-type(5) {
+    flex-basis: 60px;
+    /* border: red 2px solid;  */
+    position: relative;
+    transform: translateX(57px)
+  }
+  .menu li:nth-of-type(6) {
+     flex-basis: 5px;
+  /*   border: red 2px solid;  */
+    position: relative;
+    transform: translateX(51px)
+  }
+  .menu li:nth-of-type(7) {
+   flex-basis: 70px;
+   /*  border: red 2px solid;  */
+    position: relative;
+    transform: translateX(58px)
+  }
+
+
+.menu li a {
+  
+    font-family: ﻿Lato-Bold, ﻿AdobeInvisFont, ﻿MyriadPro-Regular;
+    font-size: 7.21379pt;
+    color: rgba(255, 255, 255, 255); 
+    text-align: justify;
+    display: flex;
+    /* flex: 1; */
+    justify-content: center; 
+    align-items: center;  
+    margin-top: 32px;
+    text-decoration: none;
+   
+    /* word-spacing:10px; */
+}
+
+
+#first-item {
+  
+    font-family: ﻿Lato-Bold, ﻿AdobeInvisFont, ﻿MyriadPro-Regular;
+    font-size: 7.21379pt;
+    color: rgba(46, 179, 152, 255); 
+     /*
+   text-align: justify;
+    display: flex;
+    justify-content: center; 
+    align-items: center;  
+    margin-top: 30px;*/
+}
+
+#separator {
+    width: 2px;
+    height: 2px;
+    border-radius: 100%;
+    background-color: #484c5f;
+    display: flex;
+    justify-content: center; 
+    align-items: center;  
+    margin-top: 30px;
+} 
+
+
+
+.logo  {
+   /*  display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto; */
+    position: absolute;
+left: 50%;
+bottom: 0%;
+top: 10px;
+transform: translateX(-50%);
+   
+}
+
+ .content {
+    top: -50px;
+    /* flex: 1 0 auto; */
+} 
+
+.hexagon-container {
+   
+    background-color: #ffffff;
+    height: 635px;
+
+}
+
+/* .figure {
+   transform: translateY(50px)
+   transform: translateX(50px);
+} */
+
+.hexagon {
+    width: 180px;
+    height: 206px;
+    margin: 7px;
+    transform: rotate(120deg);
+    overflow: hidden;
+    visibility: hidden;
+    display: inline-block;
+   
+}
+
+.hex-layer-1 {
+    width: 100%;
+    height: 100%;
+    transform: rotate(-60deg);
+    overflow: hidden;
+    visibility: hidden;
+}
+
+.hex-layer-2 {
+    width: 100%;
+    height: 100%;
+     background: #cccc;
+    transform: rotate(-60deg);
+    overflow: hidden;
+    visibility: visible;
+    background-image: url('../assets/images/Layer33@1X.png');
+    background-repeat: no-repeat;
+    background-position: 50%;
+}
+
+
+.pic1  {
+  width: 100%;
+  height: 100%;
+   background: #cccc;
+  transform: rotate(-60deg);
+  overflow: hidden;
+  visibility: visible;
```

Comment:
`visible` is the default value, remove it

Reply:
Ok